src/Http/Offer/PriceInfoValidatingRequestBodyParser.php

```diff
@@ -16,12 +16,14 @@ public function parse(ServerRequestInterface $request): ServerRequestInterface
     {
         $data = $request->getParsedBody();
 
-        if (!isset($data->priceInfo)) {
+        if (!isset($data->priceInfo) && !is_array($data)) {
```

Comment:
I think this requires a comment to indicate that either a full event or place is passed with the price info as property or either the price info as the root object.